csharp/autobuilder/Semmle.Autobuild.CSharp/StandaloneBuildRule.cs

```diff
@@ -35,6 +42,12 @@ BuildScript GetCommand(string? solution)
                     cmd.Argument("--skip-nuget");
                 }
 
+                if (!string.IsNullOrEmpty(this.dotNetPath))
```

Comment:
A couple of lines above there's `cmd.Argument("--references:.");` We're installing dotnet into the `.dotnet` folder inside the repo. Does this mean that all DLLs inside that folder will also be automatically added to the reference list?

Reply:
I believe so, assuming the logic in `AssemblyCache` traverses directories starting with `.`

Reply:
That's not great, because we'll be adding SDK assemblies, such as csc itself. Also asp.net core dlls will be added too even for projects that are not web applications.

Could we install to `Path.GetTempPath()`? In the standalone extractor we already have some logic to store files in ` Path.Combine(Path.GetTempPath(), "GitHub", ...`. Also if we go through `TemporaryDirectory` it could be easily cleaned up.

Another option would be to exclude the `.dotnet` folder from the DLL lookup, but that might not be straightforward as we do want to add some DLLs from the `.dotnet` folder.